src/components/Button/Button.test.tsx

```diff
@@ -19,7 +18,7 @@ describe('Button', () => {
   })
 
   describe('with extra className and disabled', () => {
-    const  { container } = render(<Button className={this.className} disabled={true}>Сохранить</Button>)
+    const  { container } = render(<Button className={'Profile-Button'} disabled={true}>Сохранить</Button>)
```

Comment:
Не забывай форматировать. И строку лучше сразу:
```suggestion
    const { container } = render(
      <Button className="Profile-Button" disabled={true}>
        Сохранить
      </Button>
    );
```